packages/docs-data/package.json

```diff
@@ -13,10 +13,13 @@
     "@blueprintjs/core": "^3.0.0",
     "@blueprintjs/docs-theme": "^3.0.0",
     "documentalist": "^1.4.0",
+    "better-handlebars": "github:wmeldon/better-handlebars",
     "glob": "^7.1.2",
     "highlights": "^3.1.1",
+    "language-less": "github:atom/language-less",
```

Comment:
can we find a published version of this syntax definition as well?

Reply:
i looked but it's only published to atom's `apm`, not npm.